.storybook/main.js

```diff
@@ -1,6 +1,13 @@
 const { getWebpackConfig } = require('nuxt')
 
 module.exports = {
+    stories: ['../components/**/*.stories.(js|mdx)'],
+    stories: ['../components/**/*.stories.js'],
```

Comment:
This line should be removed, `js` are handle in  `(js|mdx)`

Reply:
yeah my bad i tried to implement mdx support but not working as expected